spec.md

```diff
@@ -210,7 +212,7 @@ extmodule MyExternalModule :
   parameter y = 42
 ```
 
-An example of an externally defined module with references is:
+An example of an externally defined module with references and groups is:
 
 ```firrtl
 extmodule MyExternalModuleWithRefs :
```

Comment:
AndGroups?

Reply:
Fixed in https://github.com/chipsalliance/firrtl-spec/pull/108/commits/0a54caf039ac72960226a34c368eafeb2cc90911